superset-frontend/src/views/CRUD/welcome/SavedQueries.tsx

```diff
@@ -57,19 +64,51 @@ interface SavedQueriesProps {
   mine: Array<Query>;
 }
 
+export const CardStyles = styled.div`
+  cursor: pointer;
+  a {
+    text-decoration: none;
+  }
+  .ant-card-cover {
+    border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+    & > div {
+      height: 171px;
```

Comment:
This can be an experiment for another PR, but if this card is basically a text box, we may be able to base its height on the number of lines set elsewhere in the JS. That is, if `line-height` is pulled from the Theme variables, the height could be `numLines * lineHeight`. Anyway, something to fiddle with to make this a little more dynamic.